pkg/errors/filesystem.go

```diff
@@ -0,0 +1,19 @@
+package errors
+
+import "fmt"
+
+// ErrFilesystem wraps unexpected filesystem errors for clearer messaging.
+type ErrFilesystem struct {
```

Comment:
I don't think dependents systems need to know that CM uses a filesystem-based DB.
In general the errors should be meaningful enough for underlying systems to determine how to cope with errors, but not enough to make them crawl under the load of knowing what to do, e.g., if the filesystem of CM has issues.
For instance, CTFd and players can't do anything about CM failures with its filesystem, but logged errors can help Ops determine what is going on!